src/meta/encoding_test.go

```diff
@@ -94,3 +94,59 @@ func TestTypeEncoding(t *testing.T) {
 		}
 	}
 }
+
+func TestConstantValueDecodeEncode(t *testing.T) {
+	// encode test
+	v := ConstantValue{
+		Type:  Integer,
+		Value: int64(54),
+	}
+	b, err := v.GobEncode()
+	if err != nil {
+		t.Errorf("unexpected error \"%s\"", err)
+	}
+
+	if b[0] != byte(Integer) || b[1] != 53 || b[2] != 52 {
```

Comment:
It's OK to test just encode+decode and compare that decoded value is the same as the original value.
This test with `b[1] != 53` is quite hard to comprehend and it depends on the fact that we're storing numerical values as text (hence we have 2 ascii bytes for 54 - 53 and 52).